lessons/models.py

```diff
@@ -515,7 +515,7 @@ def standards_by_category(self):
 """
 
 
-class Activity(Orderable, CloneableMixin):
+class Activity(Internationalizable, Orderable, CloneableMixin):
```

Comment:
Similar question to above. Moving forward is there a cost to making all new things as internationaliable, or should we wait to actually need translation?